### ISSUE TITLE
Crash with Fabric API >= 0.34.0

### ISSUE THREAD

**Issue by NebelNidas** [CodedSakura/FabricWarps#1]
When using latest FabricWarps (v0.2.1) in combination with Fabric API >= 0.34.0, the game crashes.

<details>
  <summary>Crashlog</summary>

```
[23:57:27] [main/INFO]: Loading for game Minecraft 1.16.5
[23:57:28] [main/INFO]: [FabricLoader] Loading 52 mods: fabric-renderer-api-v1@0.4.2+2868a2287d, fabric-keybindings-v0@0.2.1+ca58154a7d, notenoughcrashes@3.2.0, fabricloader@0.11.3, fabric-structure-api-v1@1.1.4+ca58154a7d, fabric-containers-v0@0.1.11+9354966b7d, fabric-dimensions-v1@2.0.6+61336ba97d, fabric-object-builder-api-v1@1.9.5+c88702897d, java@11, fabric-game-rule-api-v1@1.0.6+ca58154a7d, fabric-api-base@0.3.0+c88702897d, fabric-rendering-data-attachment-v1@0.1.5+ca58154a7d, fabric-textures-v0@1.0.6+ca58154a7d, fabric-rendering-fluids-v1@0.1.13+ca58154a7d, fabric-blockrenderlayer-v1@1.1.5+ca58154a7d, fabric-lifecycle-events-v1@1.2.1+ca58154a7d, fabric-renderer-registries-v1@2.3.0+2868a2287d, fabric-loot-tables-v1@1.0.2+ca58154a7d, fabric@0.34.6+1.16, fabric-particles-v1@0.2.4+ca58154a7d, fabric-networking-v0@0.3.2+ca58154a7d, fabric-api-lookup-api-v1@1.0.0+dc716ea17d, fabric-networking-blockentity-v0@0.2.8+ca58154a7d, minecraft@1.16.5, fabric-screen-handler-api-v1@1.1.5+9354966b7d, fabric-command-api-v1@1.1.1+351679a77d, fabric-screen-api-v1@1.0.0+c045166c7d, fabric-tool-attribute-api-v1@1.2.7+8183d3957d, fabric-renderer-indigo@0.4.4+ca58154a7d, fabric-item-api-v1@1.2.1+ca58154a7d, fabric-crash-report-info-v1@0.1.3+ca58154a7d, fabric-events-interaction-v0@0.4.2+ca58154a7d, fabric-entity-events-v1@1.1.0+c88702897d, fabrictpafixtitles@0.1.2, fabric-rendering-v0@1.1.2+ca58154a7d, fabric-key-binding-api-v1@1.0.4+9354966b7d, fabric-rendering-v1@1.6.0+2868a2287d, fabric-resource-loader-v0@0.4.6+2868a2287d, fabricwarps@0.2.1, fabric-content-registries-v0@0.2.1+ca58154a7d, fabric-tag-extensions-v0@1.1.1+ca58154a7d, cardinal-components-world@2.7.9, fabric-biome-api-v1@3.1.10+3661eaa17d, fabric-commands-v0@0.2.2+ca58154a7d, fabric-registry-sync-v0@0.7.4+ca58154a7d, cardinal-components-base@2.7.9, fabric-mining-levels-v0@0.1.3+ca58154a7d, fabric-networking-api-v1@1.0.3+e3c9d0627d, fabric-events-lifecycle-v0@0.2.1+ca58154a7d, fabric-models-v0@0.3.0+bc7a746f7d, fabric-item-groups-v0@0.2.7+2868a2287d, fabric-object-builders-v0@0.7.2+ca58154a7d
[23:57:28] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8.2 Source=file:/C:/Users/User/AppData/Roaming/.minecraft/libraries/net/fabricmc/sponge-mixin/0.9.2+mixin.0.8.2/sponge-mixin-0.9.2+mixin.0.8.2.jar Service=Knot/Fabric Env=CLIENT
[23:57:36] [main/INFO]: Environment: authHost='https://authserver.mojang.com', accountsHost='https://api.mojang.com', sessionHost='https://sessionserver.mojang.com', servicesHost='https://api.minecraftservices.com', name='PROD'
[23:57:37] [main/INFO]: Setting user: User
[23:57:37] [main/INFO]: Initializing StacktraceDeobfuscator
[23:57:37] [main/INFO]: Done initializing StacktraceDeobfuscator
[23:57:38] [main/INFO]: [Indigo] Registering Indigo renderer!
[23:57:38] [main/INFO]: Backend library: LWJGL version 3.2.2 build 10
[23:57:47] [main/INFO]: Narrator library for x64 successfully loaded
[23:57:47] [main/INFO]: Reloading ResourceManager: Default, Fabric Mods (Fabric Renderer API (v1), Fabric Key Bindings (v0), Not Enough Crashes, Fabric Loader, Fabric Structure API (v1), Fabric Containers (v0), Fabric Dimensions API (v1), Fabric Object Builder API (v1), Fabric Game Rule API (v1), Fabric API Base, Fabric Rendering Data Attachment (v1), Fabric Textures (v0), Fabric Rendering Fluids (v1), Fabric BlockRenderLayer Registration (v1), Fabric Lifecycle Events (v1), Fabric Renderer Registries (v1), Fabric Loot Tables (v1), Fabric API, Fabric Particles (v1), Fabric Networking (v0), Fabric API Lookup API (v1), Fabric Networking Block Entity (v0), Fabric Screen Handler API (v1), Fabric Command API (v1), Fabric Screen API (v1), Fabric Tool Attribute API (v1), Fabric Renderer - Indigo, Fabric Item API (v1), Fabric Crash Report Info (v1), Fabric Events Interaction (v0), Fabric Entity Events (v1), Fabric Rendering (v0), Fabric Key Binding API (v1), Fabric Rendering (v1), Fabric Resource Loader (v0), Fabric Content Registries (v0), Fabric Tag Extensions (v0), Fabric Biome API (v1), Fabric Commands (v0), Fabric Registry Sync (v0), Fabric Mining Levels (v0), Fabric Networking API (v1), Fabric Events Lifecycle (v0), Fabric Models (v0), Fabric Item Groups (v0), Fabric Object Builders (v0)), Programmer Art ()
[23:57:56] [main/INFO]: OpenAL initialized.
[23:57:56] [main/INFO]: Sound engine started
[23:57:57] [main/INFO]: Created: 1024x1024x4 minecraft:textures/atlas/blocks.png-atlas
[23:57:57] [main/INFO]: Created: 256x128x4 minecraft:textures/atlas/signs.png-atlas
[23:57:57] [main/INFO]: Created: 1024x512x4 minecraft:textures/atlas/banner_patterns.png-atlas
[23:57:57] [main/INFO]: Created: 1024x512x4 minecraft:textures/atlas/shield_patterns.png-atlas
[23:57:57] [main/INFO]: Created: 256x256x4 minecraft:textures/atlas/chest.png-atlas
[23:57:57] [main/INFO]: Created: 512x256x4 minecraft:textures/atlas/beds.png-atlas
[23:57:57] [main/INFO]: Created: 512x256x4 minecraft:textures/atlas/shulker_boxes.png-atlas
[23:57:57] [main/INFO]: Created: 256x256x0 minecraft:textures/atlas/particles.png-atlas
[23:57:57] [main/INFO]: Created: 256x256x0 minecraft:textures/atlas/paintings.png-atlas
[23:57:57] [main/INFO]: Created: 256x128x0 minecraft:textures/atlas/mob_effects.png-atlas
[23:58:03] [main/WARN]: Ambiguity between arguments [teleport, destination] and [teleport, targets] with inputs: [Player, 0123, @e, dd12be42-52a9-4a91-a8a1-11c01849e498]
[23:58:03] [main/WARN]: Ambiguity between arguments [teleport, location] and [teleport, destination] with inputs: [0.1 -0.5 .9, 0 0 0]
[23:58:03] [main/WARN]: Ambiguity between arguments [teleport, location] and [teleport, targets] with inputs: [0.1 -0.5 .9, 0 0 0]
[23:58:03] [main/WARN]: Ambiguity between arguments [teleport, targets] and [teleport, destination] with inputs: [Player, 0123, dd12be42-52a9-4a91-a8a1-11c01849e498]
[23:58:03] [main/WARN]: Ambiguity between arguments [teleport, targets, location] and [teleport, targets, destination] with inputs: [0.1 -0.5 .9, 0 0 0]
[23:58:03] [main/INFO]: Reloading ResourceManager: Default, Fabric Mods (Fabric Tool Attribute API (v1))
[23:58:04] [main/INFO]: Loaded 7 recipes
[23:58:04] [main/INFO]: Loaded 927 advancements
[23:58:04] [main/INFO]: Applied 0 biome modifications to 0 of 79 new biomes in 1.804 ms
[23:58:05] [main/WARN]: Ambiguity between arguments [teleport, destination] and [teleport, targets] with inputs: [Player, 0123, @e, dd12be42-52a9-4a91-a8a1-11c01849e498]
[23:58:05] [main/WARN]: Ambiguity between arguments [teleport, location] and [teleport, destination] with inputs: [0.1 -0.5 .9, 0 0 0]
[23:58:05] [main/WARN]: Ambiguity between arguments [teleport, location] and [teleport, targets] with inputs: [0.1 -0.5 .9, 0 0 0]
[23:58:05] [main/WARN]: Ambiguity between arguments [teleport, targets] and [teleport, destination] with inputs: [Player, 0123, dd12be42-52a9-4a91-a8a1-11c01849e498]
[23:58:05] [main/WARN]: Ambiguity between arguments [teleport, targets, location] and [teleport, targets, destination] with inputs: [0.1 -0.5 .9, 0 0 0]
[23:58:05] [main/INFO]: Reloading ResourceManager: Default, Fabric Mods (Fabric Tool Attribute API (v1))
[23:58:06] [main/INFO]: Loaded 7 recipes
[23:58:06] [main/INFO]: Loaded 927 advancements
[23:58:07] [main/INFO]: Environment: authHost='https://authserver.mojang.com', accountsHost='https://api.mojang.com', sessionHost='https://sessionserver.mojang.com', servicesHost='https://api.minecraftservices.com', name='PROD'
[23:58:07] [Server thread/INFO]: Starting integrated minecraft server version 1.16.5
[23:58:07] [Server thread/INFO]: Generating keypair
[23:58:07] [Server thread/ERROR]: Encountered an unexpected exception
java.lang.ExceptionInInitializerError: null
	at Not Enough Crashes deobfuscated stack trace.(1.16.5+build.4) ~[?:?]
	at net.minecraft.world.World.mdf4807f$lambda$static$0$0(World:1233) ~[?:?]
	at net.minecraft.util.Lazy.get(Lazy:16) ~[?:?]
	at net.minecraft.world.World.handler$zzf000$initComponents(World:1240) ~[?:?]
	at net.minecraft.world.World.<init>(World:153) ~[?:?]
	at net.minecraft.server.world.ServerWorld.<init>(ServerWorld:186) ~[?:?]
	at net.minecraft.server.MinecraftServer.createWorlds(MinecraftServer:368) ~[intermediary-fabric-loader-0.11.3-1.16.5.jar:?]
	at net.minecraft.server.MinecraftServer.loadWorld(MinecraftServer:335) ~[intermediary-fabric-loader-0.11.3-1.16.5.jar:?]
	at net.minecraft.server.integrated.IntegratedServer.setupServer(IntegratedServer:67) ~[?:?]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer:645) ~[intermediary-fabric-loader-0.11.3-1.16.5.jar:?]
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer:257) ~[intermediary-fabric-loader-0.11.3-1.16.5.jar:?]
	at java.lang.Thread.run(Thread.java:834) [?:?]
Caused by: java.lang.RuntimeException: Failed to hack fabric API
	at dev.onyxstudios.cca.internal.base.ComponentsInternals.<clinit>(ComponentsInternals.java:56) ~[017fa38b-58f3-4bb0-8ae4-c79fedc8a9c4.jar:?]
	... 11 more
Caused by: java.lang.NoSuchFieldException: type
	at java.lang.Class.getDeclaredField(Class.java:2411) ~[?:?]
	at dev.onyxstudios.cca.internal.base.ComponentsInternals.<clinit>(ComponentsInternals.java:52) ~[017fa38b-58f3-4bb0-8ae4-c79fedc8a9c4.jar:?]
	... 11 more
[23:58:07] [Server thread/ERROR]: This crash report has been saved to: C:\Users\User\Documents\Minecraft\Minecraft 1.16.5 Fabric Test\crash-reports\crash-2021-05-31_23.58.07-server.txt
[23:58:07] [Server thread/INFO]: Stopping server
[23:58:07] [Server thread/INFO]: Saving players
[23:58:07] [Server thread/INFO]: Saving worlds
[23:58:07] [Server thread/ERROR]: Exception stopping the server
java.lang.NullPointerException: null
	at Not Enough Crashes deobfuscated stack trace.(1.16.5+build.4) ~[?:?]
	at net.minecraft.server.MinecraftServer.save(MinecraftServer:572) ~[intermediary-fabric-loader-0.11.3-1.16.5.jar:?]
	at net.minecraft.server.MinecraftServer.shutdown(MinecraftServer:599) ~[intermediary-fabric-loader-0.11.3-1.16.5.jar:?]
	at net.minecraft.server.integrated.IntegratedServer.shutdown(IntegratedServer:203) ~[?:?]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer:707) ~[intermediary-fabric-loader-0.11.3-1.16.5.jar:?]
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer:257) ~[intermediary-fabric-loader-0.11.3-1.16.5.jar:?]
	at java.lang.Thread.run(Thread.java:834) [?:?]
```
</details>

**Comment by NebelNidas** [comment]
Looks to be the same issue as here: https://github.com/emilyalexandra/trinkets/issues/85

Only that over there the map loading just freezes, with FabricWarps the game completely crashes 🤷‍♂️

**Comment by CodedSakura** [comment]
hmm, that looks like an issue with a dependency, aka easy to fix

**Comment by CodedSakura** [comment]
fixed in v1.0.0
